### ISSUE TITLE
ghcid FILE fails after stack build

### ISSUE THREAD

**Issue by simonmichael** [ndmitchell/ghcid#69]
I have to use `ghcid -c 'stack ghci'` in stack projects as `ghcid PATH` fails once you've done a build:
```
$ stack new a simple
$ cd a
$ ghcid src/Main.hs  # succeeds
$ stack build
$ ghcid src/Main.hs
Loading stack exec --test -- -fno-code src/Main.hs ...
Executable named -fno-code not found on path: ...
Command "stack exec --test -- -fno-code src/Main.hs" exited unexpectedly
$ rm -rf .stack-work
$ ghcid src/Main.hs # succeeds again
```

**Comment by ndmitchell** [comment]
Thanks for the report, yep, that looks pretty broken. I'll take a look shortly.

**Comment by ndmitchell** [comment]
Very nasty bug due to a single word missing - but when the word is `ghci` and the project is `ghcid` that matters a lot! Fixed and released as 0.6.4.

**Comment by simonmichael** [comment]
Much better. Thanks for this awesome tool!
